lib/msal-react/src/MsalProvider.tsx

```diff
@@ -19,6 +21,11 @@ export type MsalProviderProps = PropsWithChildren<{
 }>;
 
 export function MsalProvider({instance, children}: MsalProviderProps): React.ReactElement {
+    // Create a logger instance for msal-react with the same options as PublicClientApplication
+    const logger: Logger = useMemo(() => {
+        return instance.getLogger().clone(Constants.SKU, Constants.VERSION);
+    }, [instance]);
+
```

Comment:
Does `logger` need to be added on line 52?

Reply:
Yes, thanks! Updated useEffect dependencies throughout.